engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/repositories/new/views/new-repository.html

```diff
@@ -332,8 +332,11 @@ <h4 translate> Published Repository Information </h4>
       <div class="checkbox" ng-hide="repository.content_type === 'docker' || repository.content_type === 'ansible_collection'">
         <label>
           <input id="unprotected" name="unprotected" ng-model="repository.unprotected" type="checkbox"/>
-          <span translate>Publish via HTTP</span>
+          <span translate>Uprotected</span>
```

Comment:
it says "uprotected" instead of "unprotected" :)

Reply:
~~Udated~~ Updated! 😄